examples/thermodynamics/determination_of_constants_in_van_der_waals_equation.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+from sympy import solve, Symbol, Eq
+from symplyphysics import print_expression, Quantity, units, convert_to
+from symplyphysics.laws.thermodynamics.equations_of_state import van_der_waals_state_equation as van_der_waals_law
+
+# Example from 2.24 in https://studfile.net/preview/1772224/page:8/
+
+temperature_before = Symbol("temperature_before")
+temperature_after = Symbol("temperature_after")
+volume = Symbol("volume")
+amount_of_substance = Symbol("amount_of_substance")
+pressure_before = Symbol("pressure_before")
+pressure_after = Symbol("pressure_after")
+
+parameter_a = Symbol("parameter_a")
+parameter_b = Symbol("parameter_b")
+
+state_equation_before = van_der_waals_law.law.subs({
+    van_der_waals_law.volume: volume,
+    van_der_waals_law.temperature: temperature_before,
+    van_der_waals_law.pressure: pressure_before,
+    van_der_waals_law.bonding_forces_parameter: parameter_a,
+    van_der_waals_law.molecules_volume_parameter: parameter_b,
+    van_der_waals_law.amount_of_substance: amount_of_substance,
+    van_der_waals_law.units.molar_gas_constant: Quantity(8.31446262 * units.joules / (units.moles * units.kelvins))
+})
+
+state_equation_after = van_der_waals_law.law.subs({
+    van_der_waals_law.volume: volume,
+    van_der_waals_law.temperature: temperature_after,
+    van_der_waals_law.pressure: pressure_after,
+    van_der_waals_law.bonding_forces_parameter: parameter_a,
+    van_der_waals_law.molecules_volume_parameter: parameter_b,
+    van_der_waals_law.amount_of_substance: amount_of_substance,
+    van_der_waals_law.units.molar_gas_constant: Quantity(8.31446262 * units.joules / (units.moles * units.kelvins))
+})
+
+solved = solve((state_equation_before, state_equation_after), (parameter_a, parameter_b), dict=True)[0]
+parameter_a_value = solved[parameter_a]
+parameter_b_value = solved[parameter_b]
+
+answer1 = Eq(parameter_a, parameter_a_value)
+answer2 = Eq(parameter_b, parameter_b_value)
+print(f"Equation for parameter a:\n{print_expression(answer1)}")
+print(f"Equation for parameter b:\n{print_expression(answer2)}")
+
+parameter_a_ = parameter_a.subs({
+    temperature_before: Quantity(300 * units.kelvins),
+    temperature_after: Quantity(350 * units.kelvins),
+    volume: Quantity(0.25 * units.liters),
+    amount_of_substance: Quantity(1 * units.amount_of_substance),
+    pressure_before: Quantity(90 * units.atmospheres),
+    pressure_after: Quantity(110 * units.atmospheres),
+})
+parameter_b_ = parameter_b.subs({
+    temperature_before: Quantity(300 * units.kelvins),
+    temperature_after: Quantity(350 * units.kelvins),
+    volume: Quantity(0.25 * units.liters),
+    amount_of_substance: Quantity(1 * units.amount_of_substance),
+    pressure_before: Quantity(90 * units.atmospheres),
+    pressure_after: Quantity(110 * units.atmospheres),
```

Comment:
You have duplicate set of values. Put them is variable, like this:
```
values = {
    temperature_before: Quantity(300 * units.kelvins),
    temperature_after: Quantity(350 * units.kelvins),
}
```
And use in subs.